.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: python
 python: 2.7
 sudo: false
+# A hack to install 3.5, until Travis-CI includes it
```

Comment:
I actually favor the issue-linked TODO here. [When a strange thing appears in code, it's nice for there to be an issue tracking making it unstrange, but it's even nicer if it includes a link to that issue in its strangeness](https://github.com/google/oauth2client/pull/503#discussion_r61331868).